webrender/src/renderer.rs

```diff
@@ -4881,7 +4908,17 @@ impl<B: hal::Backend> Renderer<B> {
         }
 
         #[cfg(not(feature = "gl"))]
-        self.device.begin_render_pass(false);
+        {
+            if !target.border_segments_solid.is_empty()
```

Comment:
and here
the point being to put this logic close to the declaration of the fields, so it's easier to keep in sync

Reply:
Added it.